core/arch/arm/sm/sm_a32.S

```diff
@@ -295,9 +295,11 @@ sm_vect_table_a15:
 	write_iciallu
 	isb
 	b	1f
+END_FUNC sm_vect_table_a15
+
```

Comment:
remove empty line

Reply:
I'm sorry, it seems I missed this comment.

Reply:
I guess that not a big issue :)